_maps/snake.json

```diff
@@ -0,0 +1,27 @@
+{
+    "map_name": "NSV Snake",
+    "map_path": "map_files/Snake",
+    "map_file": [
+		"snake_lower.dmm", 
+		"snake_upper.dmm"
+	], 
+    "traits": [
+		{
+			"Up": 1,
+			"linkage": "self"
+		}, 
+		{
+			"Down": -1,
+			"linkage": "self"
+		}
+	],
+    "space_ruin_levels": -1,
+	"space_empty_levels": 0,
+        "shuttles": {
+		"cargo": "cargo_box",
+        "emergency": "emergency_goon",
+		"ferry": "ferry_kilo"
+	},
+	"mine_disable": 1,
+	"ship_type": "/obj/structure/overmap/nanotrasen/missile_cruiser/starter"
```

Comment:
Please don't use missile_cruiser/starter
Not everything needs to use this.
Please use patrol_cruiser/starter etc

Reply:
Resolved in e48947d